Update the sites content

MrMagnus opened the pull request:
Uppdate the sites images and text according to what the customer wanted.
Also update the Gemfile.lock to include json 1.8.3 instead of 1.8.1 due to crashing when executing bundle install.

Commit:
Update Gemfile.lock to include json 1.8.3

https://trello.com/c/BiCFBzaY

Commit:
Update the sites content

Uppdate the sites text and images according to what the customer wanted

https://trello.com/c/BiCFBzaY

scmx commented on .bowerrc:
```diff
@@ -1,3 +1,3 @@
-}
\ No newline at end of file
+  "directory" : "public/components"
+}
```
Changing _sass to public/components looks like an unrelated change. Was there a particular reason why this was changed together with json version?

alex-ross replied:
@scmx: @MrMagnus had some issues with permissions so I told him to add this if there wasn't a file already that defined a directory.

However. As you say this change will break stuff. But the previous one isn't correct. when the value is `/_sass` it wants to create that folder in my computers root directory 😄. I'll will reverse this change and fix the old one.

scmx commented:
Looking good

Commit:
Update gem github-pages

Updated the github-pages gem to match the version that github uses.

https://trello.com/c/BiCFBzaY

alex-ross committed:
Change bower directory for sass components
